lib/components/user/places/favorite-place-screen.js

```diff
@@ -151,7 +164,7 @@ class FavoritePlaceScreen extends Component {
                   <div>
                     {isMobileView && place && (
                       <SaveButton bsStyle='primary' type='submit'>
-                        <FormattedMessage id='common.forms.save' />
+                        {pendingSave ? <InlineLoading />:<FormattedMessage id='common.forms.save' />}
```

Comment:
Same nit here, would prefer keeping the text. You could make the text portion `visibility: hidden` to help keep the button size.

Reply:
Other buttons that follow this pattern do not keep the text.